list.js

```diff
@@ -1,18 +1,27 @@
 (function() {
-    var events = [];
+    var eventsMap = new Map();
 
-    var renderEvents = function(events) {
+    var renderAllEvents = function(eventList) {
         var markup = '';
+        eventList.forEach(function(value, key, map) {
+            markup += value.render();
```

Comment:
@nikolas thats a good point about the `eventList` param name.  I left the parameter name as `eventList` for that function because I thought it would be general enough if its changed to another kind of data structure.

The anonymous function passed into `forEach` requires a map in its signature though, so a "general enough" name doesn't save any work, and is confusing.  I'll change this too.